dask_jobqueue/core.py

```diff
@@ -87,6 +90,8 @@ class JobQueueCluster(Cluster):
     submit_command = None
     cancel_command = None
     scheduler_name = ''
+    
+    shell = False # Required for submitting jobs with LSF
```

Comment:
Ideally we wouldn't mention LSF here in the superclass.  We would say "required for submitting jobs that require full shell access" or something else like that (I don't actually understand the issue behind `shell`, so I recommend finding better language.)